src/__tests__/events.tsx

```diff
@@ -175,10 +175,11 @@ test("calling `fireEvent` directly works too", () => {
 
   fireEvent(
     button!,
-    new Event("MouseEvent", {
+    new MouseEvent("click", {
       bubbles: true,
-      cancelable: true,
-      button: 0
-    } as any)
+      cancelable: true
+    })
   );
+
+  expect(handleEvent).toHaveBeenCalledTimes(1);
```

Comment:
I think it would be better to check if we receive the correct event, as in:

```ts
  const clickEvent = new MouseEvent("click", {
    bubbles: true,
    cancelable: true
  });

  fireEvent(button!, clickEvent);

  expect(handleEvent).toHaveBeenCalledWith(clickEvent);
```

Reply:
The requested changes have been made.